cirq/circuits/circuit.py

```diff
@@ -1387,6 +1389,48 @@ def tetris_concat(
 
         return cirq.Circuit(buffer[offset : offset + n_acc])
 
+    def get_qubit_factors(self) -> List[Set['cirq.Qid']]:
```

Comment:
this may be an unfamiliar name. I don't know if people usually call them "factors". Maybe `get_indepdenent_qubit_sets`

Reply:
Done.